phabricator.py

```diff
@@ -37,8 +39,25 @@ def run(self):
         # Find the preselected branch
         git_branch = settings.get('branch')
 
-        # If no preselected branch is provided
-        if git_branch == None:
+        if git_branch is None and settings.get('use_arc_land_onto_default', False):
```

Comment:
This is starting to feel like we are creating new settings for each different setting of `branch` handling. Instead of this, can we make it optional to specify branch as a `dict` and have an option in there?

```python
{
  branch: {
    use_arc_land_onto_default: True
  }
}
```